messaging/MessagingExample/AppDelegate.m

```diff
@@ -15,15 +15,14 @@
 
 #import "AppDelegate.h"
 
-#if defined(__IPHONE_10_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
 @import UserNotifications;
-#endif
 
 // Implement UNUserNotificationCenterDelegate to receive display notification via APNS for devices
 // running iOS 10 and above.
-#if defined(__IPHONE_10_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
+#if __IOS_AVAILABLE
 @interface AppDelegate () <UNUserNotificationCenterDelegate>
 @end
+@end
```

Comment:
Please remove the extra `@end` declaration.

The `__IOS_AVAILABLE` wrapper here is probably unnecessary, since the whole AppDelegate class depends on UIKit/iOS.